config/initializers/local_config.rb.example

```diff
@@ -1,19 +1,18 @@
-# Google OAuth2 credentials
-Devise.setup do |config|
-  config.omniauth :google_oauth2, "xxx.apps.googleusercontent.com", "xxx"
+# Replace all <bracketed> items with your config info.
+
+# Google OAuth2 credentials.  Get <client ID> and <client secret> from your Google Developers MESS OAuth client.
+Devise.setup do |config
+  config.omniauth :google_oauth2, "<client ID>", "<client secret>"
 end
 
-# generate your secret key with `rake secret`
-Mess::Application.config.secret_key_base = "xxx"
+# Get a '<secret key>' by running 'rake secret'.
+Mess::Application.config.secret_key_base = "<secret key>"
 
-# The hostname of your server. Enter `localhost` for a standard development setup.
-Rails.configuration.x.host = "example.com"
+# Replace '<hostname>' with your web server.  Enter 'localhost' for a standard development setup.
+Rails.configuration.x.host = "<hostname>"
 
+# Replace '<hostname>' with your smtp server.  Enter 'localhost' for a standard development setup.
 Rails.configuration.action_mailer.smtp_settings = {
-  address: "example.com",
-  port: 587,
-  domain: "example.org",
-  authentication: :plain,
-  user_name: "user",
-  password: "xxx"
+  address: "<hostname>",
+  port: 1025
```

Comment:
The mailcatcher port 1025 config is already in development.rb file. I would leave this as it was but perhaps comment the whole thing out and say "For production environments only, uncomment and enter SMTP settings. Development environments use default mailcatcher settings." or some such.